### PR TITLE
feat(normalize-docs): transform emojis #59

### PR DESCRIPTION
**Pull request by smollweide**
#59

### CODE REVIEW

File: packages/storybook-readme/src/services/transformEmojis.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const emoji = require('node-emoji');
```
Comment (ndelangen) [inline packages/storybook-readme/src/services/transformEmojis.js]: Need to evaluate the weight, security, maintenance of this package, and whether it's included in the runtime.

File: packages/storybook-readme/src/services/transformEmojis.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const transformEmojis = doc => {
+  const resultMap = {};
+  let result;
+  while ((result = EMOJI.exec(doc)) !== null) {
+    resultMap[result] = true;
+  }
+  Object.keys(resultMap).forEach(result => {
+    doc = doc.replace(
+      new RegExp(result, 'g'),
```
Comment (ndelangen) [inline packages/storybook-readme/src/services/transformEmojis.js]: Would be nice to move this regex up to the top of the file, so there's only 1 single regex instance and it's re-used.

**Comment by ndelangen** [comment]
Thanks for this PR @smollweide !

File: packages/storybook-readme/src/services/transformEmojis.js (hunk 1 above)
Comment (ndelangen) [reply packages/storybook-readme/src/services/transformEmojis.js]: https://bundlephobia.com/result?p=node-emoji@1.8.1

Comment (ndelangen) [reply packages/storybook-readme/src/services/transformEmojis.js]: What do you think @tuchk4 ?

Comment (tuchk4) [reply packages/storybook-readme/src/services/transformEmojis.js]: There is 27Kb (minified) SON file with string-placeholder to emoji map.  
It is not possible to make it lesser. Only if restrict the used emoji :)

https://github.com/omnidan/node-emoji/blob/master/lib/emoji.json

**Comment by tuchk4** [comment]
I think there is the great chance that the same emoji declaration format `:heart:` (semicolon before and after emoji key) could be used at doc files.  We cant guarantee back compatibility with existing storybook that uses this addon. 

Maybe better to emojify only inside headings and paragraphs tags?

**Comment by ndelangen** [comment]
1 way to remove the 27kb is to do the conversion at build time, possibly in a webpack loader?

**Comment by smollweide** [comment]
Hi Folks,
what‘s the status? Is something missing? Should I change something?

Best,
Simon

**Comment by ndelangen** [comment]
@smollweide So the issue is the size of the package this adds. It isn't minor.

Merging this would add it to add storybook-readme users.
The ideal solution would be one where the transform is done via babel.

That would make it so 0 additional code is added.

**Pull request merged**